src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1269,6 +1269,36 @@ public Tizen.NUI.PropertyMap Placeholder
             }
         }
 
+        /// This will be released at Tizen.NET API Level 9, so currently this would be used as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Current Master branch is for API Level 9, 
So, It need a description and need to remove EditorBrowsable

Reply:
I removed comment of API Level 9.